src/xrCore/Threading/TaskManager.cpp

```diff
@@ -67,7 +69,7 @@ static constexpr size_t TASK_STORAGE_MASK = TASK_STORAGE_SIZE - 1;
 class FallbackTaskAllocator
 {
     std::atomic_size_t m_allocated{};
-    Task               m_storage[TASK_STORAGE_SIZE];
+    Task               m_storage[TASK_STORAGE_SIZE]{};
```

Comment:
This class works the same as `TaskAllocator` class.
Refer to the comment about that class.
```suggestion
    Task               m_storage[TASK_STORAGE_SIZE];
```